### DISCUSSION TITLE
What do you do on holidays?

### DISCUSSION THREAD

**Discussion by Araki-SH** [NAIST-SE/AttractingContributors#812] [2023 Attraction Contributions]
What do you do on days when you are free from classes and research?

Or do you escape from reports and research?

Please let us know what you do!

And enjoy your day off!

Well, the day I post this is Monday.

https://github.com/Araki-SH/AttractingContributors_What-to-do-on-holidays-.git

**Comment by shanpeng-m** [comment]
Hi, I have request a PR.
Could you please help me add something
https://github.com/NAIST-SE/AttractingContributors/discussions/804

**Comment by Gloria109** [comment]
I went skiing and i had a great trip to Europe
